vars/createRepo.groovy

```diff
@@ -0,0 +1,23 @@
+def call(String REPO_NAME, String REPO_TYPE) {
+    node('source-builder') {
+        deleteDir()
+
+        withCredentials([sshUserPrivateKey(credentialsId: 'alex-10.30.6.51', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+            sh """
+                ssh -o StrictHostKeyChecking=no -i ${KEY_PATH} ${USER}@10.30.6.51 ' \
+                    set -o errexit
+                    set -o xtrace
+
+                    pushd /srv/repo-copy
+                        wget https://raw.githubusercontent.com/vorsel/aux/master/scripts/create-repos/create_repo.sh
```

Comment:
We need to check if file exists or not as it might be placed here manually or ssh connection was broken and rm command didn't execute on previous run

Reply:
Done, fixed by using -O flag for wget while obtaning the script